create Flux Ecosystem blog post aboud Weave GitOps (#1446)
fluxcd/website: content/en/blog/2023-04-04-how-to-use-weave-gitops-as-your-flux-ui/index.md

---
author: dholbach
date: 2023-04-04 08:30:00+00:00
title: How to use Weave GitOps as your Flux UI
description: "..."
url: /blog/2023/04/how-to-use-weave-gitops-as-your-flux-ui/
tags: [ecosystem]
resources:
- src: "**.{png,jpg}"
  title: "Image #:counter"
---

Here comes the newest blog post in our [ecosystem category](/tags/ecosystem/).
One of the key reasons to rewrite Flux was to break up the former monolith
solution into separate controllers which provide distinct parts of the
functionality. This allows users to pick just the parts they need, and
integrators to very easily build on top of Flux's APIs. Today we have a very
active [Flux Ecosystem](/ecosystem/) - we very much welcome this to happen and
see it as an indicator of success.

## An introduction to Weave GitOps

Today we would like to talk about [Weave
GitOps](https://github.com/weaveworks/weave-gitops). It has
been built out in the open for about a year and brings among other
things one of the most requested additions to Flux: a UI.

{{< gallery match="wego*.png" sortOrder="desc" rowHeight="150" margins="5"
            previewType="color" embedPreview=true >}}

<br>

> With Weave GitOps you can

Reviewer: ```suggestion
With Weave GitOps you
```

> - Manage and view applications all in one place
> - Easily see your continuous deployments and what is being produced
>   via GitOps
> - Sync your latest git commits directly from the UI
> - Leverage Kubernetes RBAC to control permissions in the dashboard
> - Quickly See the health of your reconciliation deployment runtime

Reviewer: ```suggestion
- manage and view applications all in one place
- easily see your continuous deployments and what is being produced
  via GitOps
- sync your latest git commits directly from the UI
- leverage Kubernetes RBAC to control permissions in the dashboard
- quickly see the health of your reconciliation deployment runtime
```

The Weave GitOps team works very closely together with the Flux
Community - many engineers on both teams are actually colleagues.

> What you get additionally when using Weave GitOps is a frictionless way
> to get GitOps set up, called [GitOps
> Run](https://docs.gitops.weave.works/docs/gitops-run/overview/).

Reviewer: ```suggestion
In addition to the UI Weave GitOps provides a frictionless way
to get up to speed with your GitOps experience: [GitOps
Run](https://docs.gitops.weave.works/docs/gitops-run/overview/).
```
Makes the sentence a little less complicated.
